test/t/test_ip.py

```diff
@@ -31,6 +31,14 @@ def test_stats(self, completion):
     def test_neigh_state(self, completion):
         assert "stale" in completion
 
+    @pytest.mark.complete(
+        "ip neig",
+        require_cmd=True,
+        skipif="ip neighbor help 2>/dev/null; (( $? != 255 ))",
+    )
+    def test_neigh_one_completion(self, completion):
+        assert len(completion) == 1
```

Comment:
This is fine, but we could be more explicit, like
```suggestion
        assert completion == "hbor"
```
Anyway pre-approved, feel free to merge either way.

Reply:
Tried, but it doesn't work, since ip on centos7 (and presumably older OSs as well) have `neigh` in the help, so that's whats offered.